.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -24,11 +24,63 @@ labels: bug
 4. See error
 
 ### Your personal set up
-<!-- Tell us a little about the system you're using. -->
+<!--
+Tell us a little about the system you're using.
+Please include information about how you installed,
+e.g. are you using a distribution such as zero-to-jupyterhub or the-littlest-jupyterhub.
+ -->
 
  - OS:
- <!-- [e.g. linux, OSX] -->
- - Version:
- <!-- e.g. jupyterhub --version. --->
- - Configuration:
- <!-- Be careful not to share any sensitive information. --->
+ <!-- [e.g. ubuntu 20.04, macOS 11.0] -->
+ - Version(s):
+ <!-- e.g. jupyterhub --version, python --version --->
+
+<!-- For reproduction, it's useful to have the full environment. For example, the output of `pip freeze` or `conda list` --->
+
+<details>
+```
+# paste output of `pip freeze` or `conda list` here
+```
+</details>
```

Comment:
I was thinking it would be nice to have a name and description for each collapsible details section. 

Without this, it looks a bit hard to determine what is what:
![details](https://user-images.githubusercontent.com/7579677/100894964-ba13e600-34c5-11eb-984e-a199a2f9fe68.png)

What do you think? 
``````suggestion
<details>
<summary>Full environment</summary>
<!-- For reproduction, it's useful to have the full environment. For example, the output of `pip freeze` or `conda list` --->
```
# paste output of `pip freeze` or `conda list` here
```
</details>
``````